apps/small_filters/conv3x3a16.cpp

```diff
@@ -0,0 +1,61 @@
+#include "Halide.h"
+
+using namespace Halide;
+using namespace Halide::Internal;
```

Comment:
Don't (or at least shouldn't) need to pull in Halide::Internal, or declare the 'irp' below. Same for the other generators.